Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
Indeed, personal preferences such as editors or "ll" shell aliases (see below) should not be forced upon people...